### ISSUE TITLE
/f promote

### ISSUE THREAD

**Issue by teracube** [poggit-orphanage/FactionsPro#66]
FactionsPro_dev-45.phar
PocketMine-MP version 3.7.1

```

2019-03-30 [13:16:35] [Server thread/INFO]: [Commandlook] HulkTheProGamer> /f promote SamTheBanana3

2019-03-30 [13:16:35] [Server thread/CRITICAL]: Error: "Call to a member function getName() on null" (EXCEPTION) in "plugins/FactionsPro_dev-45.phar/src/FactionsPro/FactionCommands" at line 291
2019-03-30 [13:16:35] [Server thread/DEBUG]: #0 plugins/FactionsPro_dev-45.phar/src/FactionsPro/FactionMain(119): FactionsPro\FactionCommands->onCommand(pocketmine\Player object, pocketmine\command\PluginCommand object, string f, array Array())
2019-03-30 [13:16:35] [Server thread/DEBUG]: #1 src/pocketmine/command/PluginCommand(58): FactionsPro\FactionMain->onCommand(pocketmine\Player object, pocketmine\command\PluginCommand object, string f, array Array())
2019-03-30 [13:16:35] [Server thread/DEBUG]: #2 src/pocketmine/command/SimpleCommandMap(270): pocketmine\command\PluginCommand->execute(pocketmine\Player object, string f, array Array())
2019-03-30 [13:16:35] [Server thread/DEBUG]: #3 src/pocketmine/Server(2010): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\Player object, string f promote SamTheBanana3)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #4 src/pocketmine/Player(2221): pocketmine\Server->dispatchCommand(pocketmine\Player object, string f promote SamTheBanana3)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #5 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(239): pocketmine\Player->chat(string /f promote SamTheBanana3)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #6 src/pocketmine/network/mcpe/protocol/CommandRequestPacket(54): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleCommandRequest(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #7 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(109): pocketmine\network\mcpe\protocol\CommandRequestPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #8 src/pocketmine/network/mcpe/protocol/BatchPacket(119): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #9 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(109): pocketmine\network\mcpe\protocol\BatchPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #10 src/pocketmine/Player(3114): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #11 src/pocketmine/network/mcpe/RakLibInterface(170): pocketmine\Player->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #12 vendor/pocketmine/raklib/src/server/ServerHandler(99): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 83.XX.XXX.116 41776, raklib\protocol\EncapsulatedPacket object, integer 0)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #13 src/pocketmine/network/mcpe/RakLibInterface(110): raklib\server\ServerHandler->handlePacket()
2019-03-30 [13:16:35] [Server thread/DEBUG]: #14 src/pocketmine/network/mcpe/RakLibInterface(100): pocketmine\network\mcpe\RakLibInterface->process()
2019-03-30 [13:16:35] [Server thread/DEBUG]: #15 vendor/pocketmine/snooze/src/SleeperHandler(123): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
2019-03-30 [13:16:35] [Server thread/DEBUG]: #16 vendor/pocketmine/snooze/src/SleeperHandler(85): pocketmine\snooze\SleeperHandler->processNotifications()
2019-03-30 [13:16:35] [Server thread/DEBUG]: #17 src/pocketmine/Server(2328): pocketmine\snooze\SleeperHandler->sleepUntil(double 1553948194.4585)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #18 src/pocketmine/Server(2186): pocketmine\Server->tickProcessor()
2019-03-30 [13:16:35] [Server thread/DEBUG]: #19 src/pocketmine/Server(1765): pocketmine\Server->start()
2019-03-30 [13:16:35] [Server thread/DEBUG]: #20 src/pocketmine/PocketMine(250): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/teracube/, string /home/teracube/plugins/)
2019-03-30 [13:16:35] [Server thread/DEBUG]: #21 (1): require(string phar:///home/teracube/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2019-03-30 [13:16:35] [Server thread/INFO]: HulkTheProGamer [-]
2019-03-30 [13:16:35] [Server thread/INFO]: HulkTheProGamer[/83.XX.XXX.116:41776] logged out due to Internal server error

```

**Comment by Awzaw** [comment]
Should be fixed by 489437f, please test...

**Comment by teracube** [comment]
@Awzaw
solved with latest build
thanks
